### PR TITLE
[ZT] Add new client MDM IP override parameters

### PR DESCRIPTION
**Pull request by schultetwin1**
The next release of the WARP client will include new MDM parameters which allow users to change the IP address uses to talk with the API, DoH, and WARP endpoints.

This change adds documentations for those MDM parameters

### CODE REVIEW

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 1]
```diff
@@ -111,6 +111,66 @@ When the WARP client is deployed via MDM, the in-app **Send Feedback** button is
 - `https://<support.example.com>` &mdash; Use an `https://` link to open your company's internal help site.
 - `mailto:<yoursupport@example.com>` &mdash; Use a `mailto:` link to open your default mail client.
 
+### `override_api_endpoint`
+
+Overrides the IP address the WARP client will use to communicate with the client
+orchestration API. This parameter is only usable by clients with a version newer
+than 2022.10.0.0. Most IT admins should not set this setting as it will redirect
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:118]: If most admins should not use these settings, what's the use case for these override parameters?

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 2]
```diff
@@ -111,6 +111,66 @@ When the WARP client is deployed via MDM, the in-app **Send Feedback** button is
 - `https://<support.example.com>` &mdash; Use an `https://` link to open your company's internal help site.
 - `mailto:<yoursupport@example.com>` &mdash; Use a `mailto:` link to open your default mail client.
 
+### `override_api_endpoint`
+
+Overrides the IP address the WARP client will use to communicate with the client
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:116]: Where can a user see the default api_endpoint, doh_endpoint, and warp_endpoint?

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md [hunk 3]
```diff
@@ -111,6 +111,66 @@ When the WARP client is deployed via MDM, the in-app **Send Feedback** button is
...
+orchestration API. This parameter is only usable by clients with a version newer
+than 2022.10.0.0. Most IT admins should not set this setting as it will redirect
+all API traffic to a new IP. If the parameter is used, please be sure to update
+your organization's firewall to ensure this IP is allowed through.
+
+Although the IP has changed the domain name used will remain the same, meaning
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:122]: > Although the IP has changed the domain name used will remain the same, meaning this new IP address may not terminate the TLS connection.

Could you clarify what this sentence means?

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md (hunk 1 above)
Comment (kkrum) [reply content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:118]: This is primarily for Zero Trust usage of employees inside China and will require a separate endpoint the customer will need to support.

I think we should add some language like "This should be primarily used by customers who need to route traffic through 3rd party providers (ex. companies operating inside China that have negotiated path through the firewall with 3rd parties)." or something like that. We'll have blogs and more documentation coming at a later date.

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md (hunk 2 above)
Comment (kkrum) [reply content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:116]: End users won't be able to see this at all, this is purely an admin controlled thing.

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md (hunk 3 above)
Comment (kkrum) [reply content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:122]: @schultetwin1 how does this impact us at all? The only communication with these endpoints should be our daemon, which all works in this mode.

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md (hunk 2 above)
Comment (schultetwin1) [reply content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:116]: If this override is set, users will be able to see it via `warp-cli settings` which will output the override value

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md (hunk 3 above)
Comment (schultetwin1) [reply content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md:122]: The point of this sentence was to make it clear you could not redirect cloudflare's traffic to a destination not owned by Cloudflare. I'm concerned people may read this and think they can host their own server that captures WARP's API requests or DNS requests.

**Comment by schultetwin1** [comment]
Covered by #6643 so closing this one

**Pull request closed without merging**